### PR TITLE
SASS-10720 Added feature-switch flag to useBusinessDetailsStub and useEncryption

### PR DESCRIPTION
**Pull request by matthewwardcap**
### Description
Requested here - https://github.com/hmrc/app-config-qa/pull/24190

##### Before Reviewing
- [ ]  Have you pulled the branch down?
- [ ]  Have you assigned yourself to the PR?
- [ ]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [ ]  Have you addressed warnings where appropriate?
- [ ]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

**Comment by matthewwardcap** [comment]
> * Your service can now use Java 21. Please set `JavaType.OPENJDK_JRE21` in [build-jobs](https://github.com/hmrc/build-jobs/blob/HEAD/jobs/live/itsass.groovy#L77) - See confluence https://confluence.tools.tax.service.gov.uk/x/xIACM
> * `AesGCMCrypto` can be used instead of `SecureGCMCipher`, it's included with your version of `bootstrap-play`. Remove [app/utils/SecureGCMCipher.scala](https://github.com/hmrc/income-tax-calculation/blob/SASS-10720/app/utils/SecureGCMCipher.scala#L45) and replace all occurances - for more details see https://github.com/hmrc/crypto

The above has been raised as an issue in the team slack notes. I'll ask Rosie to raise a ticket for it as it seems like it could be a full 2 point ticket in terms of work

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢

File: conf/application.conf [hunk 1]
```diff
@@ -96,11 +96,11 @@ mongodb {
   timeToLive = 60minutes
 }
 
-useBusinessDetailsStub = false
-useEncryption = false
 income-tax-submission-stub-url = "https://income-tax-submission-stub.protected.mdtp:443"
 useGetCalcListIFPlatform = true
```
Comment (MJCallahanPage) [inline conf/application.conf:100]: This one needs moving too to match app-config PR

**Review by MJCallahanPage — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 1 above)
Comment (matthewwardcap) [reply conf/application.conf]: https://github.com/hmrc/income-tax-calculation/pull/110/files#diff-40b59660a23903e3ce6c42593b69e162e754b949c3fe120777db7e159a5ea529R79

**Review by basotia — APPROVED** [review]

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢

**Pull request merged**